public/js/attendance-scripts/attendance-form-add-new-person.js

```diff
@@ -107,14 +107,14 @@ function validateAge(age) {
     if(isNaN(ageNumber)) return false;
     if(typeof(ageNumber) !== "number") return false;
     //returns true if age in valid range 18-100
-    return((ageNumber >= 18) && (ageNumber <= 100));
+    return((ageNumber >= 1) && (ageNumber <= 130));
```

Comment:
Just check if `ageNumber > 0` there's no need to make up some upper value, we don't have a requirement. Same for `numChildren >= 0`. It may seem pedantic but the minimum needed here is just a positive number.